configure-wazuh.sh

```
@@ -0,0 +1,266 @@
+    read -p "Public URL (e.g., https://wazuh.company.com or https://server.com/wazuhsse) [http://localhost:$MCP_PORT]: " PUBLIC_URL
+    PUBLIC_URL=${PUBLIC_URL:-http://localhost:$MCP_PORT}
+
+    # Extract base path from URL if present
+    if [[ "$PUBLIC_URL" == */* ]] && [[ "$PUBLIC_URL" != *"://"*"/"*"/"* ]]; then
+        BASE_PATH=""
+    elif [[ "$PUBLIC_URL" =~ ^https?://[^/]+(/.*) ]]; then
+        BASE_PATH="${BASH_REMATCH[1]}"
+        PUBLIC_URL_BASE="${PUBLIC_URL%$BASE_PATH}"
+    else
+        BASE_PATH=""
+        PUBLIC_URL_BASE="$PUBLIC_URL"
```

Shellcheck (0.10.0)

[warning] PUBLIC_URL_BASE appears unused. Verify use (or export if used externally).

(SC2034)